install_config/install/docker_registry.adoc

```diff
@@ -10,8 +10,8 @@
 
 toc::[]
 
+[[overview-of-deploying-a-docker-registry]]
```

Comment:
A topic's "Overview" is the exception to the rule about creating unique anchor IDs. We'd just link to the topic itself w/ no anchor if trying to link to its Overview.